packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -299,20 +294,27 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                     {(isSortingOrFiltering && props.paging ? rowsPagination : rows).map((row, index) => {
                         prepareRow(row);
                         return (
-                            <div {...row.getRowProps()} key={`row_${index}`} className="tr">
+                            <tr {...row.getRowProps()} key={`row_${index}`} className="tr">
                                 {row.cells.map(cell => cell.render("Cell"))}
-                                {props.columnsHidable && (
-                                    <div className="td column-selector" style={{ width: columnSelectorWidth }} />
-                                )}
-                            </div>
+                                {props.columnsHidable && <td className="td column-selector" />}
+                            </tr>
                         );
                     })}
                 </InfiniteBody>
-                <div className="tfoot">
-                    {props.pagingPosition === "bottom" && pagination}
-                    {props.footerWidgets}
-                </div>
-            </div>
+            </table>
+            {props.pagingPosition === "bottom" && pagination}
```

Comment:
Could a div wrapper around a custom header and footer be useful from styling perspective?

Reply:
Good point, im gonna re-add it